### ISSUE TITLE
Don't ship ebtables-save / ebtables-restore with CoreOS

### ISSUE THREAD

**Issue by kayrus** [coreos/bugs#1105]
``` sh
cat /etc/lsb-release 
DISTRIB_ID=CoreOS
DISTRIB_RELEASE=933.0.0
DISTRIB_CODENAME="Red Dog"
DISTRIB_DESCRIPTION="CoreOS 933.0.0"
```

``` sh
$ head -1 /usr/sbin/ebtables-save
#!/usr/bin/perl -w
```

``` sh
$ /usr/sbin/ebtables-save
-bash: /usr/sbin/ebtables-save: /usr/bin/perl: bad interpreter: No such file or directory
```

Looks like we have to fix [ebtables ebuild](https://github.com/coreos/portage-stable/blob/master/net-firewall/ebtables/ebtables-2.0.10.4-r1.ebuild).

Probably, we can use ebtables-save written in bash: https://bugzilla.redhat.com/show_bug.cgi?id=746040

**Comment by bgilbert** [comment]
ebtables-save is no longer shipped in Container Linux, and ebtables-restore is not written in Perl.

**Comment by bgilbert** [comment]
Fixed by coreos/coreos-overlay#2352.
